### ISSUE TITLE
Email as User id

### ISSUE THREAD

**Issue by awead** [samvera/hydra-role-management#21]
User ids are integers, however, when deleting users from roles, an email string is used:

```
http://localhost:3000/roles/3/users/awead@aic-dot-org
```

As opposed to the integer id of the user:

```
http://localhost:3000/roles/3/users/1
```

Is there a Devise configuration I'm missing?

**Comment by kerchner** [comment]
@awead is this causing you a stack trace when trying to remove a user from a role?  It is for me.  I've been looking into a fix, but nothing yet.

**Comment by awead** [comment]
@kerchner I overrode the edit view to get that to work:

https://github.com/aic-collections/aicdams-lakeshore/blob/master/app/views/roles/edit.html.erb

**Comment by jrgriffiniii** [comment]
Resolved by #32
